Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -682,6 +682,33 @@ TEST_CASE(test_jpeg2000_tag_tree)
         EXPECT_EQ(1u, MUST(tree.read_value(2, 1, read_bit, next_layer)));
         EXPECT_EQ(index, 7u); // Didn't change!
     }
+
```

Comment:
Commit message nitpick: The title seems _too_ generic to me. I'm sure it is a very interesting test case, but the important part is that we (presumably) now handle massively lopsided trees correctly.

Reply:
is "generic" the word you meant? If so, I'm not sure what you're saying. I reworded it a bit assuming that you meant "specific" instead. LMK if that's what you wanted :)

Reply:
I did mean to say "generic" (because the original title didn't mention what the point of the test case and of removing the `VERIFY` was; that information is nice to read even when looking at just the oneline logs), although maybe it could also be considered "very specific, but focused on the wrong part".

The new commit title is very much fine.